welcome.md

```diff
@@ -0,0 +1,10 @@
+Welcome to the Software Carpentry email list
+============================================
+
+This is a shared list for Software and Data Carpentry community members.
+
+Software and Data Carpentry are community organizations. We value the involvement of every one in this community - learners, instructors, hosts, developers, steering committee members and staff. We are committed to creating a friendly and respectful place for learning, teaching and contributing. All participants in Software Carpentry events or communications are expected to show respect and courtesy to others.
+
+To make clear what is expected, everyone participating in Software and Data Carpentry activities is required to conform to the following Code of Conduct. This code of conduct applies to all spaces managed by Software Carpentry, including, but not limited to, workshops, email lists, online forums and on GitHub.
```

Comment:
“the following Code of Conduct” should be “our [Code of Conduct](https://software-carpentry.org/conduct)”.  And if we go with caps there, I suggest the following sentence start with “This Code of Conduct applies…”.